.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```

Comment:
This table is also a little overdo. not required i think @sambhav228

Reply:
ISSUE TEMPLATE updated

Reply:
@sambhav228 you haven't updated the template yet. There are no new changes